### DISCUSSION TITLE
How to force KafkaConnector to wait for `Ready` KafkaConnect cluster?

### DISCUSSION THREAD

**Discussion by lbennett-stacki** [strimzi/strimzi-kafka-operator#4133] [Q&A] [answered]
I want to `kubectl apply` my `KafkaConnect` cluster and `KafkaConnector` declarations at the same time. Unfortunately when doing this, the connect cluster doesn't register the connector and it is not running and it must be re-applied when the connect cluster is running.

**Comment by scholzj** [comment, marked as answer]
There is not way to wait apart from using a script which would actually wait. But it should not be needed. The Connector should be eventually created in one of the next reconciliations after the Connect is deployed. So I do not think there should be any need to do anything to the Connector. If that is not happening, please share the custom resources including their status and the logs from the Cluster Operator.

**Reply by lbennett-stacki** [reply]
Perfect! Thanks @scholzj!
